common/api-review/performance-exp.api.md

```diff
@@ -5,18 +5,47 @@
 ```ts
 
 import { FirebaseApp } from '@firebase/app-exp';
-import { FirebasePerformance } from '@firebase/performance-types-exp';
-import { PerformanceSettings } from '@firebase/performance-types-exp';
-import { PerformanceTrace } from '@firebase/performance-types-exp';
 
-export { FirebasePerformance }
+// @public (undocumented)
+export interface FirebasePerformance {
+    dataCollectionEnabled: boolean;
+    instrumentationEnabled: boolean;
+}
 
 // @public
-export function getPerformance(app: FirebaseApp, settings?: PerformanceSettings): FirebasePerformance;
+export function getPerformance(app: FirebaseApp): FirebasePerformance;
 
-export { PerformanceSettings }
-
-export { PerformanceTrace }
+// @public
+export function initializePerformance(app: FirebaseApp, settings?: PerformanceSettings): FirebasePerformance;
+
+// @public (undocumented)

```

Comment:
Should probably add documentation comments, I guess in a follow-up PR.

Reply:
@alikn Can we add missing documentation in a separate PR?

Reply:
SGTM.